src/Notepads/Package.targets

```diff
@@ -69,6 +69,17 @@
     </ItemGroup>
   </Target>
   
+  <Target Name="AssignAppCenterSecret" BeforeTargets="BeforeBuild" Condition=" $(AppCenterSecret) != '' ">
```

Comment:
@aleks-ivanov Don't put this target in `Package.targets`, put it in the project file instead. The reason I created this separate file for targets is because of #523 and these common targets will be both imported by UWP project and packaging project. Since the `AssignAppCenterSecret` target is project specific it doesn't make any sense to put it in a separate file.

Reply:
@soumyamahunt understood, I will move it back to the project file 🙂